packages/core/src/payment/strategies/stripe-upe/stripe-upe.ts

```diff
@@ -209,8 +221,24 @@ interface ShippingDefaultValues {
     };
 }
 
+export enum Mode {
```

Comment:
We should have a more descriptive name for this.

Reply:
Thank you for the comment, I have changed the name for something more descriptive and added comments on how to use it.